knowledge/faq.rst

```diff
@@ -8,5 +8,57 @@ FAQ
     There is a great community behind Conan with users helping each other in `Cpplang Slack`_.
     Please join us in the ``#conan`` channel!
 
+Troubleshooting
+---------------
+
+ERROR: Missing prebuilt package
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When installing packages (with :command:`conan install` or :command:`conan create`) it is possible
+that you get an error like the following one:
+
+.. code-block:: text
+
+    ERROR: Missing binary: zlib/1.2.11:b1d267f77ddd5d10d06d2ecf5a6bc433fbb7eeed
+
+    zlib/1.2.11: WARN: Can't find a 'zlib/1.2.11' package binary 'b1d267f77ddd5d10d06d2ecf5a6bc433fbb7eeed' for the configuration:
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=apple-clang
+    compiler.cppstd=gnu11
+    compiler.libcxx=libc++
+    compiler.version=14
+    os=Macos
+    [options]
+    fPIC=True
+    shared=False
+
+    ERROR: Missing prebuilt package for 'zlib/1.2.11'
+    Check the available packages using 'conan list zlib/1.2.11:* -r=remote'
```

Comment:
I am missing the point of checking for available packages here. if we already know, it's not available, what does it benefit? e.g. I need package for apple-clang 14, and there is a package for apple-clang 11, may I somehow force usage of existing package?

Reply:
I guess that maybe you don't care about that setting and you are ok using another setting that you have binaries available for instead of building a new one for your configuration

Reply:
yes, but then you would need to run it via `-s:h zlib:compiler.version=11`, right?
I suppose it's not that obvious and deserves some explanation